### DISCUSSION TITLE
通过【配置订阅】生成的配置文件中没有机场提供的 proxies

### DISCUSSION THREAD

**Discussion by mikeguo99** [vernesong/OpenClash#3717] [Q&A] [answered]
<img width="939" alt="image" src="https://github.com/vernesong/OpenClash/assets/111490790/47303e35-5eed-4607-ab1e-947f60041ec6">

生成的配置文件不对，那个报错的 proxy group 确实没有 proxies字段。更奇怪的是机场提供的所有proxies都没有出现在最终的 yaml 文件中。

<img width="889" alt="image" src="https://github.com/vernesong/OpenClash/assets/111490790/068877c6-8938-4b48-ac59-4effc33a84e8">

<img width="881" alt="image" src="https://github.com/vernesong/OpenClash/assets/111490790/cc6fb187-acb4-4d65-9b36-744ff4844fa9">

**Comment by ghost** [comment, marked as answer]
这是一键生成吧
再确认下
1. openclash的版本
2. 一键生成下几个清除全点一遍、取消勾选一键生成配置文件和保留配置
3. 重新订阅或上传配置文件
4. 一键生成读取配置
5. 检查一键生成页面读取到的和你原订阅或上传的配置是否一致
6. 重新生成一键配置，看是否缺失

如果还是生成失败，发bug的issue吧，前几天V大修过一次，可能还是有问题。

目前判断你这个问题其实就是勾选了保留配置的原因（包括你那个custom digioce），但话还是说全，省的来回发

**Reply by mikeguo99** [reply]
一键生成页面的加载真的超级慢

**Reply by mikeguo99** [reply]
太棒了，“一键生成下几个清除全点一遍、取消勾选一键生成配置文件和保留配置” 这么做之后重新添加配置就可以用了。
👍🏻👍🏻👍🏻👍🏻👍🏻👍🏻
